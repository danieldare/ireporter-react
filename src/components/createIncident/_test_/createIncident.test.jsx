import React from 'react';
import { mount } from 'enzyme';
import { MemoryRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import CreateIncident from '../createIncident';
import { mockStore } from '../../../../test/setupTests.js';

const props = {
  post: {
    error: 'segun',
    isCreating: false,
    success: false,
    login: jest.fn()
  },
  profile: {
    posts: ''
  },
  match: {
    url: ''
  }
};

describe('<CreateIncident /> rendering', () => {
  it('should match snapshot', () => {
    const store = mockStore(props);
    const wrapper = mount(
      <Provider store={store}>
        <Router>
          <CreateIncident match={{ url: '' }} />
        </Router>
      </Provider>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
