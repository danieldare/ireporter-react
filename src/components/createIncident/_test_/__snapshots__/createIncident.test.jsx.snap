// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateIncident /> rendering should match snapshot 1`] = `ReactWrapper {}`;
